src/main/java/com/spotify/docker/client/messages/NetworkSettings.java

```diff
@@ -70,6 +72,46 @@
   @JsonProperty("Networks")
   public abstract ImmutableMap<String, AttachedNetwork> networks();
 
+  @Nullable
+  @JsonProperty("EndpointID")
+  public abstract String endpointId();
+
+  @Nullable
+  @JsonProperty("SandboxID")
+  public abstract String sandboxId();
+
+  @Nullable
+  @JsonProperty("SandboxKey")
+  public abstract String sandboxKey();
+
+  @Nullable
+  @JsonProperty("HairpinMode")
+  public abstract Boolean hairpinMode();
+
+  @Nullable
+  @JsonProperty("LinkLocalIPv6Address")
+  public abstract String linkLocalIpv6Address();
+
+  @Nullable
+  @JsonProperty("LinkLocalIPv6PrefixLen")
+  public abstract Integer linkLocalIpv6PrefixLen();
+
+  @Nullable
+  @JsonProperty("GlobalIPv6Address")
+  @SuppressFBWarnings(value = {"NM_CONFUSING"},
+          justification = "Conforming to docker API")
+  public abstract String globalIpv6Address();
+
+  @Nullable
+  @JsonProperty("GlobalIPv6PrefixLen")
+  @SuppressFBWarnings(value = {"NM_CONFUSING"},
+          justification = "Conforming to docker API")
+  public abstract Integer globalIpv6PrefixLen();
```

Comment:
I didn't notice when I wrote my new code that it was different from any existing code. To make everything consistent I could do one of two things:
1. Change my new method names to conform to the existing ones; i.e. all `IP`. Pro: easy, Con: `IPv6` is explicitly called out as incorrect camel case in the [Google Java Syle Guide](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case).
2. All `Ip`. I would make new `Ip` methods for each existing `IP` and deprecate the old ones. Pro: "correct" style. Con: Old methods still exist (at least for a while).